tests/scripts/run-xunit-perf.cmd

```diff
@@ -15,6 +15,7 @@ setlocal
   set TEST_CONFIG=Release
   set IS_SCENARIO_TEST=
   set USAGE_DISPLAYED=
+  set UPLOAD_TO_BENCHVIEW=0
```

Comment:
Could you please rename this to something else? Currently, it has the same name of the label.